src/table/table.scss

```diff
@@ -155,6 +156,12 @@
   }
 }
 
+@mixin iui-table-header-actions-container {
```

Comment:
Also make sure this is applied in `information-panel.html` and `file-upload.html`

Reply:
Yeah, good catch.. firstly want to verify, if we go this way.

Reply:
done